x-pack/filebeat/input/entityanalytics/provider/jamf/jamf.go

```diff
@@ -350,7 +351,7 @@ func (p *jamfInput) runFullSync(inputCtx v2.Context, store *kvstore.Store, clien
 // runIncrementalUpdate will run an incremental update. The process is similar
 // to full synchronization, except only users which have changed (newly
 // discovered, modified, or deleted) will be published.
```

Comment:
> only users which have changed (newly discovered, modified, or deleted) will be published

I don't see deletes being handled anywhere.

In `doFetchComputers` it will always fetch everything and return whatever is new or modified compared to the store (unless its a fullSync, in which case it returns nothing).

I don't see any mention of deletes in the [`GET /api/preview/computers` documentation](https://developer.jamf.com/jamf-pro/reference/get_preview-computers).